co-log/src/Colog/Message.hs

```diff
@@ -227,19 +227,19 @@ fmtRichMessageDefault RichMessage{..} = do
     maybeUtcTime  <- extractField $ TM.lookup @"utcTime"  richMessageMap
     pure $ formatRichMessage maybeThreadId maybeUtcTime richMessageMsg
   where
-    formatRichMessage :: Maybe ThreadId -> Maybe UTCTime -> Message -> Text
+    formatRichMessage :: Maybe ThreadId -> Maybe C.Time -> Message -> Text
     formatRichMessage (maybe "" showThreadId -> thread) (maybe "" showTime -> time) Message{..} =
         showSeverity messageSeverity
      <> time
      <> showSourceLoc messageStack
      <> thread
      <> messageText
 
-    showTime :: UTCTime -> Text
-    showTime t = square $ T.pack $
-          formatTime defaultTimeLocale "%H:%M:%S." t
-       ++ take 3 (formatTime defaultTimeLocale "%q" t)
-       ++ formatTime defaultTimeLocale " %e %b %Y %Z" t
+    showTime :: C.Time -> Text
+    showTime t = square $ T.pack $ show $ C.builder_DmyHMS timePrecision datetimeFormat (C.timeToDatetime t)
+        where
+          timePrecision = C.SubsecondPrecisionFixed 3
+          datetimeFormat = C.DatetimeFormat (Just ' ') (Just ' ') (Just ':')
```

Comment:
According to our style guide, 3 lines above shifted 2 spaces left :slightly_smiling_face: